Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -996,10 +1007,19 @@ WI.NetworkManager = class NetworkManager extends WI.Object
                     requestId,
                     content: revision.content,
                     base64Encoded: !!revision.base64Encoded,
-                    mimeType: revision.mimeType ?? undefined,
+                    mimeType: revision.mimeType ?? "text/plain",
                     status: !isNaN(localResource.statusCode) ? localResource.statusCode : 200,
-                    statusText: !isNaN(localResource.statusCode) ? (localResource.statusText ?? "") : WI.HTTPUtilities.statusTextForStatusCode(200),
-                    headers: localResource.responseHeaders,
+                    statusText: (function() {
+                        if (!isNaN(localResource.statusCode)) {
+                            if (localResource.statusText ?? false)
```

Comment:
Shouldn't `localResource.statusText` itself be enough here? I think this could also be simplified down to `return localResource.statusText ?? WI.HTTPUtilities.statusTextForStatusCode(localResource.statusCode)`

Reply:
I was trying to match existing behavior, but I kinda agree in that there's not really a reason for us to be able to provide an arbitrary `statusText` even if there's no `statusCode`.

I think the logic was that the `statusCode` is kinda more important than the `statusText`, so if the former is not provided then we kinda dropped the latter on the floor.